### PR TITLE
Add week view

### PR DESCRIPTION
**Pull request by bach5000**
Should solve #34.
Changes:
* A week view!
* Week view toolbar button and a menu item
* When a day is selected on the calendar, all events in the same week are shown.
* All events shown in week view will contain dates as well as time, to minimize confusion.
* Events that are "All Day" will be shown with the date in week view.

### CODE REVIEW

File: src/MainWindow.cpp [hunk 1]
```diff
@@ -201,7 +207,7 @@ MainWindow::_InitInterface()
 	fMainView = new MainView();
 
 	fMenuBar = new BMenuBar("MenuBar");
-
+
```
Comment (humdingerb) [inline src/MainWindow.cpp]: Stray tab.

File: src/DayView.cpp [hunk 2]
```diff
@@ -116,7 +121,14 @@ DayView::MessageReceived(BMessage* message)
 
 			break;
 		}
-
+		case kDayView: case kWeekView:
```
Comment (waddlesplash) [inline src/DayView.cpp]: Put the second `case` on its own line

File: src/DayView.cpp [hunk 3]
```diff
@@ -116,7 +121,14 @@ DayView::MessageReceived(BMessage* message)
...
+		{
+			mode = message->what;
+			Window()->LockLooper();
```
Comment (waddlesplash) [inline src/DayView.cpp]: You are currently inside the window, so why does `LockLooper` need to be called...?

**Comment by bach5000** [comment]
Thanks, @waddlesplash!

**Pull request merged**